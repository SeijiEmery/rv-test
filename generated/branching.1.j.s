j l1
    addi t1, t1, 50
l1:
    addi t1, t1, -10
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
