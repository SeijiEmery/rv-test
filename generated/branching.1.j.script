load /x 0 generated/branching.1.j.hex
writereg 6 22
setpc 0
run 1004
readreg 6