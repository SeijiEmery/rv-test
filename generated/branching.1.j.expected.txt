tests/branching.test.s:12
R6 = 12
